docs/presentation/Content/PropertySpecification.md

```diff
@@ -83,6 +83,7 @@ This attribute controls whether the particular property is present in the result
 - Omitted or `undefined`: property visibility is controlled by the ECSchema.
 - `true`: property is made visible. **Warning:** this will automatically hide all other properties of the same class. If this behavior is not desirable, set [`doNotHideOtherPropertiesOnDisplayOverride` attribute](#attribute-donothideotherpropertiesondisplayoverride) to `true`.
 - `false`: property is made hidden.
+- The value can be set using ECExpression. **Warning:** this will automatically hide all other properties of the same class. If this behavior is not desirable, set [`doNotHideOtherPropertiesOnDisplayOverride` attribute](#attribute-donothideotherpropertiesondisplayoverride) to `true`.
```

Comment:
- ...set using **an** ECExpression.
- I think *ECExpression* should be a link that points to a section in `Content/ECExpressions.md`. The page has no section that describes symbols available for this expression, though, so you'll need to create one.
- ...this will automatically hide all other properties of the same class, **no matter what the expression evaluates to**.